install/running_install.adoc

```diff
@@ -154,20 +154,23 @@ system can run out of files to open and the playbook will fail.
 
 To run the RPM-based installer:
 
-. Run the *_prequisites.yml_* playbook. This must be run only once before
-deploying a new cluster. Use the following command, specifying `-i` if your
-inventory file located somewhere other than *_/etc/ansible/hosts_*:
+. Run the *_prequisites.yml_* playbook. This playbook installs required software
+packages, if any, and modifies the container runtimes. Unless you need to
+configure the container runtimes, run this playbook only one time, before you
```

Comment:
@kalexand-rh Do we need to explain why the user might need to configure the container runtimes?

Reply:
@mburke5678, not unless we usually go into detail about why they'd need to change their Docker settings.